proto/manifest.proto

```diff
@@ -65,6 +67,13 @@ message Resource {
     // Ads stores one or more alternate data streams for the target resource.
     repeated ADSEntry ads = 13;
 
+    // Timestamps.
+    // btime (birthtime) is available on Darwin, FreeBSD, NetBSD, and Windows.
+    // When btime is unknown, it SHOULD be set to zero.
```

Comment:
Why not omit the field instead of setting it to zero? I believe these fields are nullable.